src/misc-data/filesystem.ts

```diff
@@ -1284,3 +1286,6 @@ export const files: File[] = [
     ]}
   ]}
 ];
+export function getFiles(): File[] {
+    return rawData;
```

Comment:
To make the objects unique, the function should create them every time:
(lambda syntax also works fine)

```ts
export function getFiles(): File[] {
  return [
      {type: 'file', id: nextId(), name: 'LICENSE.md', size: 636, time: new Date('2019-12-18T15:06:11')},
      {type: 'file', id: nextId(), name: 'README.md', size: 4460, time: new Date('2019-12-18T15:06:11')},
      ...
   ]
}
```

Reply:
Would prefer it more deterministic. I was trying different syntax in order to determine the best.

Reply:
Returning a new instance of the array and the objects inside it is the deterministic option. Otherwise all consuming modules will get the same instance, regardless of the syntax in use.